x-pack/platform/plugins/shared/lens/public/datasources/form_based/to_esql.ts

```diff
@@ -145,10 +145,13 @@ export function getESQLForLayer(
 
     if (wrapInFilter || wrapInTimeFilter) {
       if (wrapInFilter) {
-        if (col.filter?.language === 'kquery') {
+        if (col.filter?.language === 'kuery') {
```

Comment:
can you remove the outer `if (wrapInFilter || wrapInTimeFilter) {` it doesn't seems to be useful at all.
Does the `kquery` language wrong all that time?